llama-index-core/llama_index/core/llms/llm.py

```diff
@@ -55,6 +57,18 @@
 
 dispatcher = instrument.get_dispatcher(__name__)
 
+if TYPE_CHECKING:
+    from llama_index.core.chat_engine.types import AgentChatResponse
+    from llama_index.core.tools.types import BaseTool
+
+
+class ToolSelection(BaseModel):
```

Comment:
wait where is this being used?

Reply:
Oh whoops, that's a leftover, let's remove